server/routes/api/messages.js

```diff
@@ -36,11 +66,37 @@ router.post("/", async (req, res, next) => {
       senderId,
       text,
       conversationId: conversation.id,
+      isRead,
     });
     res.json({ message, sender });
+    return;
   } catch (error) {
     next(error);
   }
 });
 
+router.put("/read", (req, res, next) => {
+  if (!req.user) {
+    return res.sendStatus(401);
+  }
+  const { otherUserId, conversationId } = req.body;
+  //update messages' read status to true if correct senderID and conversationID
+  Message.update(
+    { isRead: true },
+    {
+      where: {
+        senderId: otherUserId,
+        conversationId: conversationId,
+        isRead: false,
+      },
+    }
+  )
+    .then(function (rowsUpdated) {
+      res.json(rowsUpdated);
+    })
```

Comment:
I would changing this to an arrow function, since you're using them everywhere else in the app 🙂